src/main/java/se/liu/ida/hefquin/engine/federation/catalog/FederationDescriptionReader.java

```diff
@@ -71,6 +71,15 @@ public FederationCatalog parseFedDescr( final Model fd ) {
 
 			// Check the type of interface
 			if ( ifaceType.equals(FD.SPARQLEndpointInterface) ) {
+
+				int numberOfEndpointAddresses = iface.listProperties(FD.endpointAddress).toList().size();
+				if (numberOfEndpointAddresses == 0){
+					throw new IllegalArgumentException("EndpointAddress is required!");
+				}
+				if (numberOfEndpointAddresses > 1){
+					throw new IllegalArgumentException("More Than One EndpointAddress!");
```

Comment:
Please replace `EndpointAddress` by `endpointAddress`.

Additionally, please mention "SPARQL endpoint" in both of these messages.

Reply:
I changed all exception messages to be more readable.